src/app/utility/calanderization/calanderization-chart/calanderization-chart.component.html

```diff
@@ -0,0 +1,2 @@
+
+<div #monthlyMeterDataBarChart id="monthlyMeterDataBarChart"></div>
```

Comment:
Should we name this monthlyMeterDataChart since it has line options?

Reply:
Yup!